src/main/steps/task-list/utils.ts

```diff
@@ -126,3 +129,38 @@ export const getAdoptionCertificateDetailsStatus = (userCase: CaseWithId): Secti
     ? SectionStatus.NOT_STARTED
     : SectionStatus.IN_PROGRESS;
 };
+
+export const getBirthMotherDetailsStatus = (userCase: CaseWithId): SectionStatus => {
+  const names = userCase.birthMotherFirstNames && userCase.birthMotherLastNames;
+  const stillAlive = userCase.birthMotherStillAlive;
+
+  if (stillAlive === YesNoNotsure.NO) {
+    return names ? SectionStatus.COMPLETED : SectionStatus.IN_PROGRESS;
+  } else if (stillAlive === YesNoNotsure.NOT_SURE) {
+    const notAliveReason = userCase.birthMotherNotAliveReason;
+    return names && notAliveReason ? SectionStatus.COMPLETED : SectionStatus.IN_PROGRESS;
+  } else if (stillAlive === YesNoNotsure.YES) {
+    const nationality = userCase.birthMotherNationality;
+    const nationalities = userCase.birthMotherAdditionalNationalities;
+    //TODO consider not sure selection
```

Comment:
Please resolve this TODO

Reply:
hang on.. I need to handle this scenario..

Reply:
handled it now.. please take a look